src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -827,12 +827,12 @@ private static void fillModulesFromFilefiltersPackage() {
      * Fill short-to-full module names map with modules from filters package.
      */
     private static void fillModulesFromFiltersPackage() {
-        NAME_TO_FULL_MODULE_NAME.put("CsvFilter",
-                BASE_PACKAGE + ".filters.CsvFilter");
-        NAME_TO_FULL_MODULE_NAME.put("IntMatchFilter",
-                BASE_PACKAGE + ".filters.IntMatchFilter");
-        NAME_TO_FULL_MODULE_NAME.put("IntRangeFilter",
-                BASE_PACKAGE + ".filters.IntRangeFilter");
+        NAME_TO_FULL_MODULE_NAME.put("CsvFilterElement",
+                BASE_PACKAGE + ".filters.CsvFilterElement");
+        NAME_TO_FULL_MODULE_NAME.put("IntMatchFilterElement",
+                BASE_PACKAGE + ".filters.IntMatchFilterElement");
+        NAME_TO_FULL_MODULE_NAME.put("IntRangeFilterElement",
+                BASE_PACKAGE + ".filters.IntRangeFilterElement");
```

Comment:
These should not be listed here. Most are package-private and they don't extend AutomaticBean.
I will make another PR after this one to remove them.